app/models/post.rb

```diff
@@ -646,6 +646,10 @@ def full_url
     "#{Discourse.base_url}#{url}"
   end
 
+  def relative_url
```

Comment:
Probably would just add a model spec for this too using the `set_subfolder` helper, then manually checking it's what you expect e.g. `expect(post.relative_url).to eq("/forum/t/blah/123")`